metric/system/numcpu/cpu_windows.go

```diff
@@ -20,19 +20,15 @@ package numcpu
 import (
 	"fmt"
 
-	"github.com/elastic/gosigar/sys/windows"
+	"golang.org/x/sys/windows"
 )
 
 // getCPU implements NumCPU on windows
 // For now, this is a bit of a hack that just asks for per-CPU performance data, and reports the CPU count
 func getCPU() (int, bool, error) {
-
-	// get per-cpu data
-	cpus, err := windows.NtQuerySystemProcessorPerformanceInformation()
-	if err != nil {
-		return -1, false, fmt.Errorf("syscall NtQuerySystemProcessorPerformanceInformation failed: %w", err)
+	numCpu := windows.GetActiveProcessorCount(0xffff)
```

Comment:
Came here to comment on the missing param. Here you can do `windows.GetActiveProcessorCount(windows.ALL_PROCESSOR_GROUPS)`.